### PR TITLE
Adds support for Windows phone 8.0 (scan method only)

### PR DESCRIPTION
**Pull request by vladimir-kotikov**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -15,8 +15,6 @@
         <clobbers target="cordova.plugins.barcodeScanner" />
     </js-module>
 
-    <asset src="www/barcodescanner.js" target="barcodescanner.js" />
```
Comment (vladimir-kotikov) [inline plugin.xml:18]: Not required since barcodescanner.js already added via `<js-module>` element.

**Comment by purplecabbage** [comment]
Note that WP8 is still NOT supported, and also that build.phonegap.com does NOT support windows8 so this is only useful to people who install it via the phonegap-cli or cordova-cli

**Comment by sgrebnov** [comment]
Not sure why this PR was not accepted. Jesse, could you please clarify this.

**Comment by purplecabbage** [comment]
Reopened, oops,I thought it was just closed by the other one.  I have reopened it, however it will not auto merge.
Sorry, I missed some of the windows8 vs windows phone 8 portions between these 2 pulls ...

**Comment by sgrebnov** [comment]
Thank you, Jesse - I'll do rebase so that it could be merged.

**Comment by vladimir-kotikov** [comment]
@purplecabbage , i've rebased this onto master.

**Pull request merged**